src/main/java/com/epam/esm/utils/GiftCertificateDtoMapper.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.esm.utils;
+
+import com.epam.esm.dto.GiftCertificateDto;
+import com.epam.esm.entity.GiftCertificate;
+import com.epam.esm.entity.Tag;
+import org.springframework.stereotype.Component;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+
+@Component
+public class GiftCertificateDtoMapper {
+
+    public GiftCertificateDto map(GiftCertificate giftCertificate) {
+        GiftCertificateDto giftCertificateDto = new GiftCertificateDto(giftCertificate.getId(),
```

Comment:
which pattern do we use in case a constructor has lot of parameters?

Reply:
We use Builder pattern

Reply:
@a1exanddrovich please note that we'll need to fix this in the next module.

Reply:
We'll fix it